### PR TITLE
fix: case-insensitive curator comment + fix logger.warning statement

### PR DESCRIPTION
**Pull request by seasidesparrow**
modified:   adsdocmatch/oracle_util.py

Fixes Issue #22

### CODE REVIEW

**Comment by seasidesparrow** [comment]
This PR also adds a method to access the `/cleanup` endpoint of oracle_service, along with unit tests.

File: run.py [hunk 1]
```diff
@@ -188,6 +195,12 @@ def main():
             else:
                 logger.error("Both parameters are need to add matched bibcodes to oracle: file path (-mf) and the source to apply (-as).")
 
+        elif args.cleanup_oracle:
+            try:
+                status = OracleUtil().cleanup_db()
```
Comment (golnazads) [inline run.py:200]: I would get the message returned from oracle and display it. There are three types of cleanup details are here https://github.com/adsabs/oracle_service/blob/master/oraclesrv/views.py#L382

Comment (seasidesparrow) [reply run.py:200]: I refactored this to return `response.text` and modified the tests accordingly

Comment (golnazads) [reply run.py:200]: Matt, did not like the field `message`, just renamed it to `details` in oracle, I think makes more sense. If you would please make the change here. Also for testing, here is the unittest in oracle if you want to see what messages are brought back from cleanup https://github.com/adsabs/oracle_service/blob/master/oraclesrv/tests/unittests/test_oracle_service.py#L762.

Comment (golnazads) [reply run.py:200]: Great job. Thank you.

**Pull request merged**